src/utils/createSASjsInstance.ts

```diff
@@ -0,0 +1,6 @@
+import SASjs, { SASjsConfig } from '@sasjs/adapter/node'
+
+export const createSASjsInstance = (config: Partial<SASjsConfig>): SASjs => {
+  console.log('actual called')
```

Comment:
console log statement should be removed.

Reply:
Good catch.